setup.md: clarify usage of conda pip and OSs (#184)
carpentries-incubator/deep-learning-intro: setup.md

        press enter to prepend Anaconda to your `PATH` (this makes the Anaconda
        distribution the default Python).

## Installing the required packages

> [Conda](https://docs.conda.io/projects/conda/en/latest/) is the package management system associated with Anaconda and runs on Windows, macOS and Linux. 

Reviewer: ```suggestion
[Conda](https://docs.conda.io/projects/conda/en/latest/) is the package management system associated with [Anaconda](https://anaconda.org) and runs on Windows, macOS and Linux. 
```

> If you have installed Anaconda successfully, Conda should be already available in your system and work regardless of the operating system.

Reviewer: The above sentence is rather explicit to separate conda from anaconda. This sentences mixes the two again. It would be cool to clear this out. Reading this naively, a learner can be confused.

Author: It is unclear which sentence the 'above sentence' is and which other sentence mixed up concepts that were clearly separated. Please refer to line numbers.

Perhaps you prefer a formulation for line 76 like

> Conda should already be available in your system once you installed Anaconda successfully. Conda thus works regardless of the operating system.

Reviewer: Sounds good: 
```suggestion
Conda should already be available in your system once you installed Anaconda successfully. Conda thus works regardless of the operating system.
```

Make sure you have an up-to-date version of Conda running.

> See [these instructions](https://docs.anaconda.com/anaconda/install/update-version/) for updating Conda. 

Reviewer: ```suggestion
See [these instructions](https://docs.anaconda.com/anaconda/install/update-version/) for updating Conda if reqired. 
```

Author: Grammatically I would prefer:  
`See [these instructions](https://docs.anaconda.com/anaconda/install/update-version/) for updating Conda if required.`

Reviewer: Well spotted, I updated the suggestion accordingly.

{: .callout}

Open a terminal and type the command:
~~~
conda install tensorflow seaborn scikit-learn pandas
~~~
{: .source}  

> Note that installing Tensorflow causes Keras to be installed too.

Reviewer: This phrase does not reflect reality and mixes terms, think keras as an API rather than a library.
```suggestion
Note that by installing a modern version of Tensorflow, you have access to the `keras` module in it.
```

Author: I would prefer a more compact formulation like:  
> Note that modern versions of Tensorflow make Keras available as a module.

Reviewer: Fair enough, sounds like a plan. Please use this or directly commit it by virtue of:

```suggestion
Note that modern versions of Tensorflow make Keras available as a module.
```

### Troubleshooting for Windows
It is possible that Windows users will run into version conflicts. If you are on Windows and get 
errors running the command, you can try installing the packages using pip:

~~~
pip install tensorflow>=2.5 seaborn scikit-learn pandas
~~~

> {: .source}

Reviewer: I am confused by this line. When did this API change come into our jekyll dependencies?

Author: I admit confusion as well. The intention was to have the code lines treated as instructions for a generic command-line interpreter rather than for a Bash terminal.  

Perhaps this confusion, if applicable, is also a reason why the current Code Check fails, although the corresponding error explanation is empty:

![image](https://user-images.githubusercontent.com/21219611/152650786-2928a3ff-75cf-4fe1-a8cd-5234f16a5431.png)

Reviewer 2: The code check error is probably resolved if you merge master into this branch, because it looks like the error that was fixed in PR #192

Reviewer 2: Anyway, the code checker only checks the Python code. 
Jekyll does render the blocks with `{. source}` correctly so I think that's fine.

[pip](https://pip.pypa.io/en/stable/) is the package management system for Python software packages.

> It should have been installed with Python and run regardless of your operating system too.

Reviewer: ```suggestion
It is integrated with your local python installation and run regardless of your operating system too.
```

Reviewer: Cool, I changed the suggestion in this fashion.
